shim/src/epoll/epoll.c

```diff
@@ -48,12 +50,25 @@ void epoll_table_init(void)
             epoll_table[i].events[j].prev_ev = INVALID_EV;
         }
     }
+
+    epoll_table[SEND_EPFD].used = 1;
+    epoll_table[SEND_EPFD].head = 0;
+    epoll_table[SEND_EPFD].tail = 0;
+    for (int i = 0; i < MAX_EVENTS; i++)
+    {
+        epoll_table[SEND_EPFD].events[i].used = i == 0? 1 : 0;
+        epoll_table[SEND_EPFD].events[i].id = i;
+        epoll_table[SEND_EPFD].events[i].qt = -1;
+        epoll_table[SEND_EPFD].events[i].next_ev = INVALID_EV;
+        epoll_table[SEND_EPFD].events[i].prev_ev = INVALID_EV;
+    }
```

Comment:
This logic is similar to the logic just above. Could we unify that in a single function?